### PR TITLE
Fix cap deploy on stage

### PR DESCRIPTION
**Pull request by meara**
# Release Notes

TECH TASK: Continue work on reorganizing secrets/settings.

### CODE REVIEW

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -12,7 +12,7 @@
   # The secret key used by Devise. Devise uses this key to generate
   # random tokens. Changing this key will render invalid all existing
   # confirmation, reset password and unlock tokens in the database.
-  config.secret_key = Settings.devise.secret_key
+  config.secret_key = Rails.application.secrets.devise.secret_key
```
Comment (meara) [inline config/initializers/devise.rb]: This is clearly broken when the devise key does not exist, but I'm not sure what makes sense to do here. Should this key always exist?

File: app/controllers/api/order_details_controller.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@ class Api::OrderDetailsController < ApplicationController
   respond_to :json
   rescue_from ActiveRecord::RecordNotFound, with: :order_detail_not_found
 
-  http_basic_authenticate_with name: Settings.api.basic_auth_name, password: Settings.api.basic_auth_password
+  http_basic_authenticate_with name: Rails.application.secrets.api["basic_auth_name"], password: Rails.application.secrets.api["basic_auth_password"]
```
Comment (jhanggi) [inline app/controllers/api/order_details_controller.rb]: Just a warning, in Rails 5.1, the keys become symbols. http://guides.rubyonrails.org/upgrading_ruby_on_rails.html#application-secrets-now-loaded-with-all-keys-as-symbols

File: config/settings/stage.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+email:
+  from: "nucore-noreply@nucore.stage.tablexi.com"
+  fake:
+    enabled: true
+    to: ['jason@tablexi.com', "meara@tablexi.com"]
```
Comment (jhanggi) [inline config/settings/stage.yml]: Let's use double quotes here for consistency.

File: app/controllers/api/order_details_controller.rb (hunk 2 above)
Comment (meara) [reply app/controllers/api/order_details_controller.rb]: Ah okay, will make sure to update that when we move to 5.1.

**Pull request merged**